doc/cygnus-ngsi/installation_and_administration_guide/ngsi_agent_conf.md

```diff
@@ -376,7 +376,7 @@ cygnus-ngsi.sinks.cartodb-sink.channel = cartodb-channel
 # absolute path to the CartoDB file containing the mapping between FIWARE service/CartoDB usernames and CartoDB API Keys
 cygnus-ngsi.sinks.cartodb-sink.keys_conf_file = /usr/cygnus/conf/cartodb_keys.conf
 # if true the latitude and longitude values are exchanged, false otherwise
-#cygnus-ngsi.sinks.cartodb-sink.flip_coordinates = true
+#cygnus-ngsi.sinks.cartodb-sink.swap_coordinates = true
```

Comment:
Fixed in https://github.com/telefonicaid/fiware-cygnus/pull/1329/commits/423f020dace40fa1f12f7ec25758fe3fcc3aee13